src/BigData/BigData.jl

```diff
@@ -10,8 +10,9 @@ include("services/FileDataStore.jl")
 
 export AbstractDataStore
 
-export AbstractBigDataEntry, GeneralBigDataEntry, MongodbBigDataEntry, FileBigDataEntry
+export AbstractBigDataEntry, GeneralDataEntry, MongodbDataEntry, FileDataEntry
```

Comment:
Could you this change to AbstractDataEntry?

Reply:
Hi. Yes, definitely.  I changed it broke a bunch of stuff and then changed it back for the commit.

Reply:
:+1:

Reply:
Any suggestions on how to deprecate an Abstract type?